kubernetes/services/metrics-server.yaml

```diff
@@ -0,0 +1,208 @@
+# Metrics Server has specific requirements for cluster and network configuration. These requirements aren't the default for all cluster distributions. Please ensure that your cluster distribution supports these requirements before using Metrics Server:
+
+# 1. Metrics Server must be reachable from kube-apiserver by container IP address (or node IP if hostNetwork is enabled).
+
+# 2. The kube-apiserver must enable an aggregation layer.
+
+# 3. Nodes must have Webhook authentication and authorization enabled.
+
+# 4. Kubelet certificate needs to be signed by cluster Certificate Authority (or disable certificate validation by passing --kubelet-insecure-tls to Metrics Server)
+
+# 5. Container runtime must implement a container metrics RPCs (or have cAdvisor support)
+
+# Please look at the link https://github.com/kubernetes-sigs/metrics-server#requirements for reference
+
+# Latest Metrics Server release can be installed by running:
+# kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    k8s-app: metrics-server
+  name: metrics-server
+  namespace: kube-system
+---
```

Comment:
@kevin-zhonghao Kevin, where are those metrics stored? Is it possible to visualize those metrics somethere?

Reply:
@xieus Metrics Server collects resource metrics from Kubelets and exposes them in Kubernetes apiserver through Metrics API for use by Horizontal Pod Autoscaler. It is not used to store any data, it is more likely an API to get current resource usage status

Reply:
@kevin-zhonghao @xieus Is that similar to Elastic's Meatricbeat? https://www.elastic.co/blog/kubernetes-observability-tutorial-k8s-metrics-collection-and-analysis

Reply:
@cj-chung A little similar. In general, we don't use metrics-server as monitoring solution or as a source of monitoring solution metrics. Currently it is just used by HPA.